tests/test_graphical_units/test_threed.py

```diff
@@ -76,6 +76,22 @@ def construct(self):
         self.add(axes)
 
 
+class MovingVerticesTest(ThreeDScene):
+    def construct(self):
+        self.set_camera_orientation(phi=75 * DEGREES, theta=30 * DEGREES)
+        vertices = [1, 2, 3, 4]
+        edges = [(1, 2), (2, 3), (3, 4), (1, 3), (1, 4)]
+        g = Graph(vertices, edges)
```

Comment:
It's not really important, but I would probably specify `vertex_type=Dot3D` if the vertices actually have 3d coordinates.

Reply:
I've changed the vertex type.